static/app/views/monitors/components/monitorCheckIns.tsx

```diff
@@ -201,3 +205,16 @@ const Timestamp = styled(DateTime)`
 const StyledShortId = styled(ShortId)`
   justify-content: flex-start;
 `;
+
+const RowPlaceholder = styled('div')`
+  grid-column: 1 / -1;
+  padding: ${space(1)};
+
+  &:not(:last-child) {
+    border-bottom: solid 1px ${p => p.theme.innerBorder};
+  }
+`;
+
+const RowPlaceholderInner = styled(Placeholder)`
+  border-radius: ${p => p.theme.borderRadius};
+`;
```

Comment:
Should we be consistent and just not have a border radius, and decide if we want to just add this to placeholders?

Reply:
Good point — removed the radius for now. I think most placeholders should have a border radius, but haven't checked to see if it makes sense everywhere.